src/pages/patientView/PatientViewPage.tsx

```diff
@@ -73,24 +75,19 @@ export interface IPatientViewPageProps {
     clinicalDataStatus?: RequestStatus;
 }
 
-export interface PatientViewUrlParams extends QueryParams{
-    studyId:string;
-    caseId?:string;
-    sampleId?:string;
-}
-
 @inject('routing', 'appStore')
 @observer
 export default class PatientViewPage extends React.Component<IPatientViewPageProps, {}> {
 
     @observable private mutationTableColumnVisibility: {[columnId: string]: boolean}|undefined;
     @observable private cnaTableColumnVisibility: {[columnId: string]: boolean}|undefined;
+    private urlWrapper:PatientViewURLWrapper;
 
     constructor(props: IPatientViewPageProps) {
 
         super(props);
 
-
+        this.urlWrapper = new PatientViewURLWrapper(props.routing);
```

Comment:
if we're going to put in urlWrapper, maybe we should go ahead and refactor the rest of page?